site/src/sphinx/server-grpc.rst

```diff
@@ -135,7 +136,12 @@ you build a :api:`GrpcService`:
                               .allowRequestMethods(HttpMethod.POST) // Allow POST method.
                               // Allow Content-type and X-GRPC-WEB headers.
                               .allowRequestHeaders(HttpHeaderNames.CONTENT_TYPE,
-                                                   HttpHeaderNames.of("X-GRPC-WEB"));
+                                                   HttpHeaderNames.of("X-GRPC-WEB"))
```

Comment:
I'm curious if we need to add this to `GrpcHeaderNames`. @anuraaga, what do you think?

Reply:
I hadn't actually heard of that header until now, but it's not in the spec and is being removed from the improbable implementation

https://github.com/improbable-eng/grpc-web/issues/325

We can keep it here until `@improbable/grpc-web` updates though but after that we'd want to remove it from here too I think.

Tracking issue is this one I think - https://github.com/improbable-eng/grpc-web/issues/346

Reply:
I see. Then let's leave it as it is for now and consider removing it in the future.